lib/handler.js

```diff
@@ -86,6 +86,8 @@ export const today = () => getDayAsISOString(new Date())
  * @param {(import('pg').Pool, import('./typings').Filter) => Promise<object[]>} fetchStatsFn
  */
 const getStatsWithFilterAndCaching = async (pathname, searchParams, res, pgPool, fetchStatsFn) => {
+  res.setHeader('access-control-allow-origin', '*')
```

Comment:
I believe browsers disallow `*`, and we need to return a hostname instead. I propose to ditch this feature until we can do it proper, with tests etc

Reply:
IIUC the docs in [MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS), `*` is allowed only for requests with no credentials. I think we should be fine here.

> I propose to ditch this feature until we can do it proper, with tests etc

Agreed.